pages/features/_slug.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <PageComponents
+    endpoint="features"
+    :params="{ 'filters[slug][$eqi]': slug }"
```

Comment:
What is $eqi

Reply:
So It is used to filter the records. eqi(equal case insensitive)